modules/runtime/nuxeo-runtime-test/src/test/java/org/nuxeo/runtime/management/counters/TestCounters.java

```diff
@@ -35,6 +36,8 @@
 @RunWith(FeaturesRunner.class)
 @Deploy("org.nuxeo.runtime.management")
 @Features(RuntimeFeature.class)
+@Deprecated(since = "11.4")
+@Ignore("NXP-28012")
```

Comment:
Why not just removing it?

Reply:
simply because the code is still there, you may want to unignore to test it.

Reply:
Will we? 😂